src/main/java/io/jenkins/plugins/kubernetes/NoDelayProvisionerStrategy.java

```diff
@@ -71,6 +73,11 @@
             }
         }
         if (availableCapacity >= currentDemand) {
+            NodeProvisioner nodeProvisioner = label.nodeProvisioner;
+            if (availableCapacity > 0 && nodeProvisioner != null) {
+                LOGGER.log(Level.FINE, "Suggesting NodeProvisioner review");
+                Timer.get().schedule(nodeProvisioner::suggestReviewNow, 1L, TimeUnit.SECONDS);
+            }
```

Comment:
nodeProvisioner is always non-null if label is non-null.

```suggestion
            if (availableCapacity > 0 && label != null) {
                LOGGER.log(Level.FINE, "Suggesting NodeProvisioner review");
                Timer.get().schedule(label.nodeProvisioner::suggestReviewNow, 1L, TimeUnit.SECONDS);
            }
```

`availableCapacity > 0` is not useful IMO. It would be more relevant to capture the number of nodes provisioned in the current round and check if it is `>0`.